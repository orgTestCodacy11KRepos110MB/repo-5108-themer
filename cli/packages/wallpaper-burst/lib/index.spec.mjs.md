# Snapshot report for `lib/index.spec.mjs`

The actual snapshot is saved in `index.spec.mjs.snap`.

Generated by [AVA](https://avajs.dev).

## themer "burst" wallpaper lists output files

> Snapshot 1

    `␊
    Files generated:␊
    ␊
    * \`themer-wallpaper-burst-dark-1000x1000.png\`␊
    * \`themer-wallpaper-burst-light-1000x1000.png\`␊
    `
