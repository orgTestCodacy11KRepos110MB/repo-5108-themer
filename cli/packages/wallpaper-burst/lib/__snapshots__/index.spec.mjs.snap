// Vitest Snapshot v1

exports[`themer "burst" wallpaper > should list output files 1`] = `
"
Files generated:

* \`themer-wallpaper-burst-dark-1000x1000.png\`
* \`themer-wallpaper-burst-light-1000x1000.png\`
"
`;
